src/scenes/NftDetailPage/NftDetailPage.tsx

```diff
@@ -27,12 +31,21 @@ type Props = {
 function NftDetailPage({ nftId }: Props) {
   const { query } = useRouter();
 
+  const screenWidth = useBreakpoint();
+  const isMobileScreen = screenWidth === size.mobileLarge || screenWidth === size.mobile;
+
   const username = window.location.pathname.split('/')[1];
-  const collectionId = query.collectionId;
+  const collectionId = query.collectionId as string;
   // TODO: Should refactor to utilize navigation context instead of session storage
   const isFromCollectionPage =
     globalThis?.sessionStorage?.getItem('prevPage') === `/${username}/${collectionId}`;
 
+  const collectionNfts = useCollectionById({ id: collectionId })?.nfts ?? [];
+  const collectionIndexOfNft = collectionNfts?.findIndex((nft) => nft.id === nftId) ?? 0;
+
+  const prevNftId = collectionNfts[collectionIndexOfNft - 1]?.id ?? '';
+  const nextNftId = collectionNfts[collectionIndexOfNft + 1]?.id ?? '';
```

Comment:
1. Checks to find collection, or if null returns an empty array
2. Finds index of current NFT, or returns 0
3. Assigns prev/next ID to be the surrounding elements if they exist, or empty string if not
4. Empty string prevents render of arrows

Reply:
nit: instead of defaulting to empty string, i think it would be more explicit to default to `null`